containers/install-ansible-container.yml

```diff
@@ -12,13 +12,8 @@
         dest: .tmp/ansible-container
         update: yes
 
-    - name: 'Install python3-devel'
-      become: true
-      yum:
-        name: python3-devel
-
     - name: 'Install ansible-container'
       become: true
-      command: 'pip3 install -e .[docker,openshift]'
+      command: 'pip install -e .[docker,openshift] --upgrade'
```

Comment:
I tried to use the native pip module but couldn't get it to work. Have you tried it? Another thing I tried was to just ```pip install -e https://github.com/ansible/ansible-container.git[docker,openshift]``` but don't recall why it didn't work. I think it was because it always upgraded but with ```--upgrade``` it might be redundant.

Reply:
The native pip module? As in pip whatever version of python pip ends up pointed at?

Reply:
https://docs.ansible.com/ansible/latest/pip_module.html